src/pages/Criar-templates.jsx

```diff
@@ -0,0 +1,12 @@
+import { Box } from "@mui/material";
+import MonacoEditor from "../components/MonacoEditor";
```

Comment:
@EloiMatos parece que ficou faltando o diretório `templatesComponents` no roteamento do import. Pode fazer esse ajuste? 

```suggestion
import MonacoEditor from "../components/templatesComponents/MonacoEditor";
```